src/app/components/FoodCard/index.tsx

```diff
@@ -33,15 +33,23 @@ export default function FoodCard({
         <p className={styles.foodCardPricesWrapper}>
           {discountedPrice ? (
             <span className={`${styles.foodCardDiscountPrice}`}>
-              R$ {discountedPrice}
+              {discountedPrice.toLocaleString(`pt-BR`, {
+                minimumFractionDigits: 2,
+                style: `currency`,
+                currency: `BRL`,
+              })}
```

Comment:
```suggestion
              {textToBrazilCurrency(discountedPrice)}
```
You can transform this into a utils function, since you used this function twice in this file. 

To do this just create a file named utils in the same file and add this function

```ts
const priceToBrazilCurrency(price: number) => price.toLocaleString(`pt-BR`, {
                minimumFractionDigits: 2,
                style: `currency`,
                currency: `BRL`,
              })
```

Reply:
I think this function could be inside the utils folder in src, because another file can use this same implemetetion